tests/manager/manager-executeBulkWrite_error-007.phpt

```diff
@@ -0,0 +1,34 @@
+--TEST--
+MongoDB\Driver\Manager::executeBulkWrite() should not issue warning before exception
+--SKIPIF--
+<?php require __DIR__ . "/../utils/basic-skipif.inc"; ?>
+--FILE--
+<?php
+require_once __DIR__ . "/../utils/basic.inc";
+
+$bulk = new MongoDB\Driver\BulkWrite;
+$bulk->insert(['x' => 1]);
+
+// Invalid host cannot be resolved
+$manager = new MongoDB\Driver\Manager('mongodb://invalid.host:27017', ['serverSelectionTimeoutMS' => 1]);
+
+echo throws(function() use ($manager, $bulk) {
+    $manager->executeBulkWrite(NS, $bulk);
+}, 'MongoDB\Driver\Exception\RuntimeException'), "\n";
+
+// Valid host refuses connection
+$manager = new MongoDB\Driver\Manager('mongodb://localhost:54321', ['serverSelectionTimeoutMS' => 1]);
+
+echo throws(function() use ($manager, $bulk) {
+    $manager->executeBulkWrite(NS, $bulk);
+}, 'MongoDB\Driver\Exception\RuntimeException'), "\n";
+
+?>
+===DONE===
+<?php exit(0); ?>
+--EXPECT--
```

Comment:
On another note, these should be Connection exceptions I think, and not Runtime (for both of us).